### PR TITLE
roughly reflect formats/mechanisms from draft-ietf-openpgp-crypto-refresh

### PR DESCRIPTION
**Pull request by hko-s**

### CODE REVIEW

File: STATUS.md [hunk 1]
```diff
@@ -4,28 +4,40 @@ Symbols:
...
 - [ ] Cleartext Signature Framework
-- [x] Paket Generation
-- [x] Signatures
+- [x] Signatures (v4)
   - [x] Validation
   - [x] Generation
-- [x] Encryption
-- [x] Decryption
-- [x] Keys
+- [ ] Signatures (v6)
```
Comment (dignifiedquire) [inline STATUS.md:24]: I actually don't know, is there a "v5 signature"?

Comment (hko-s) [reply STATUS.md:24]: draft-koch-librepgp specifies at least the following: a "v5 key" format, "v5 signature", v5 SKESK,  and a new packet type 20 "OCB Encrypted Data Packet" (for LibrePGP's flavor of AEAD encryption)

my idea for this PR was to only mention draft-koch-librepgp at the top, and leave it at that, for now.

**Review by dignifiedquire — APPROVED** [review]

**Comment by dignifiedquire** [comment]
can you rebase please, so we can get this in

**Comment by hko-s** [comment]
> can you rebase please, so we can get this in

done

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**